v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/JdbcIoWrapper.java

```diff
@@ -220,6 +221,21 @@ private static ImmutableList<TableConfig> autoInferTableConfigs(
     return tableConfigsBuilder.build();
   }
 
+  static ImmutableList<String> getTablesToMigrate(
+      ImmutableList<String> configTables, ImmutableList<String> discoveredTables) {
+    List<String> tables = null;
+    if (configTables.isEmpty()) {
+      tables = discoveredTables;
+    } else {
+      tables =
+          configTables.stream()
+              .filter(t -> discoveredTables.contains(t))
```

Comment:
[super nit] It might help with debugging, if we add a log in the filter lambda to indicate something on the lines of  `skipping table %s, as it was not discovered on the source.`

Reply:
there is a logger right after this which prints out the table list for that purpose.